src/Exercise6.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const data = [
+  "🐶 dog",
+  "🐱 cat",
+  "🐔 chicken",
+  "🐮 cow",
+  "🐏 sheep",
+  "🏇 horse",
+];
+
+const DataHolder = (props) => (
+  <div className="border border-solid border-gray-500 p-5">{props.value}</div>
+);
+
+const Exercise6 = () => {
+  return (
```

Comment:
I recommend using the wrap all your components in the components folder to improve your folder structure. No need to mix the components in one file as you did here by having `DataHolder` & `Exercise6` in the same file.

Reply:
if I do this now, it will change all the links which other reviewers are using